### ISSUE TITLE
getting a syntax error when running grunt-saucelabs task

### ISSUE THREAD

**Issue by akonsu** [jmreidy/sauce-tunnel#27]
here is the error, would you be able to help, please?
```
Running "saucelabs-mocha:all" (saucelabs-mocha) task
=> Starting Tunnel to Sauce Labs
>> /home/.../grunt-saucelabs/node_modules/sauce-tunnel/vendor/linux/bin/sc: 1: /home/.../node_modules/grunt-saucelabs/node_modules/sauce-tunnel/vendor/linux/bin/sc: Syntax error: "(" unexpected
=> Stopping Tunnel to Sauce Labs
```

**Comment by woollybogger** [comment]
This should have fixed by #36.
Can anyone you verify this? @ponelat or @akonsu?

**Comment by ponelat** [comment]
Yup, with the 32bit binary this issue is resolved. @jmreidy, @akonsu you can close.
